msu/utils/globals.nut

```diff
@@ -127,3 +127,13 @@
 		_object = _object.get();
 	return typeof _object == "table" && "_release_hook_DO_NOT_delete_it_" in _object;
 }
+
+::MSU.new <- function( _script, _function = null )
```

Comment:
why can function be null? this seems antithetical to the purpose of this PR

Reply:
My thought to allow it to be null was so that  `::MSU.new` can be used in place of vanilla `::new` without causing any issues.

Reply:
I disagree, I don't think that's necessary and I don't think we should handle that, if someone really needs a null equivalent for some unknown reason they can pass an empty function